index.html

```diff
@@ -0,0 +1,146 @@
+        </ul>
+      </div>
+    </div>
+    <main class="main-content">
+      <section id="main">
+        <div>
+          <h1>"Hello! Tech World"</h1>
+          <h2>WEB SUMMIT 2023</h2>
+          <p id="introduction">
+            Although times may be uncertain, one thing
+            remains true. There is a simple power in people
+            coming together.
+          </p>
+          <h3>2023.07.24(MON) ~ 28(FRI)</h3>
+          <p>ENNERCARE CENTRE, TORONTO - CANADA.</p>
+        </div>
+      </section>
+      <section id="programs">
+        <h2>Main Program</h2>
+        <hr class="solid" />
+        <div id="all-programs">
+          <article class="program-cards">
+            <img src="images/programs/program1.png" alt="lecture" />
+            <h2>Lecture</h2>
+            <p>
+              Listen to speakers from various countries about the messages of
+              sharing and opening
+            </p>
+          </article>
+          <article class="program-cards">
+            <img src="images/programs/program2.png" alt="Exhibition" />
+            <h2>Exhibition</h2>
+            <p>
+              Listens to speakers from various countries about the messages of
+              sharing and opening
+            </p>
+          </article>
+          <article class="program-cards">
+            <img src="images/programs/program3.png" alt="Forum" />
+            <h2>Forum</h2>
+            <p>
+              Have the time to share your thoughts and opinions with experts
+              for each topic.
+            </p>
+          </article>
+          <article class="program-cards">
+            <img src="images/programs/program4.png" alt="Workshop" />
+            <h2>Workshop</h2>
+            <p>
+              Listen to speakers from various countries about the messages of
+              sharing and opening
+            </p>
+          </article>
+          <article class="program-cards">
+            <img src="images/programs/program5.png" alt="ignite" />
+            <h2>Collision Ignite</h2>
+            <p>
+              Get opportunities to network with Collision affiliates
+              around the World, also after the summit.
+            </p>
+          </article>
+        </div>
+        <button type="button" id="see-button">SEE THE WHOLE PROGRAM</button>
+        <button type="button" id="book-button">BOOK TICKETS</button>
+      </section>
+      <section id="speakers">
+        <h2>Meet our Speakers</h2>
+        <hr class="solid" />
+        <div class="top-speakers"></div>
+        <button type="button" id="more-button">MORE</button>
+      </section>
+      <section id="home-partners">
+        <h2>Partners</h2>
+        <hr class="solid" />
+        <div id="partner-img">
+          <img src="images/partners/logo-accenture.png" alt="accenture" />
+          <img src="images/partners/cisco-logo.png" alt="cisco" />
+          <img src="images/partners/kpmg-logo.png" alt="kpmg" />
+        </div>
```

Comment:
Please ensure that the pages should look almost identical to the original design (the margins should be the same as in the original design and the fonts).

Your Implementation | Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/164411637-cfb9e7ae-36a8-4327-ae6d-03d2148ad592.png) | ![image](https://user-images.githubusercontent.com/10332499/164411763-44ffe10a-0edd-4636-8a00-9337d733bf9a.png)

Reply:
resolved